.github/workflows/deploy.yml

```diff
@@ -0,0 +1,58 @@
+name: Deploy
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  bintray-upload:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
+    - name: Run with Gradle
+      env:
+        BINTRAY_USER: ${{ secrets.BintrayUser }}
+        BINTRAY_KEY: ${{ secrets.BintrayKey }}
+      run: ./gradlew --no-daemon bintrayUpload -PBINTRAY_USER=$BINTRAY_USER -PBINTRAY_KEY=$BINTRAY_KEY
+
+  github-release:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
+    - name: Run with Gradle
+      env:
+        GITHUB_TOKEN: ${{ secrets.GithubToken }}
+      run: ./gradlew --no-daemon githubRelease -PGITHUB_TOKEN=$GITHUB_TOKEN
+
+  pages:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
+    - name: Run with Gradle
+      run: ./gradlew --no-daemon dokkaGitlabPages
+    - name: Upload artifacts
```

Comment:
We should automate docs hosting. Use a step which looks something like this instead:
```yaml
      - uses: crazy-max/ghaction-github-pages@v2
        with:
          build_dir: public
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

This means you should also update the README's docs link from `https://neelkamath.gitlab.io/kwikipedia/` to `https://neelkamath.github.io/kwikipedia/`.